dogpush/dogpush.py

```diff
@@ -96,25 +97,25 @@ def _canonical_monitor(original, default_team=None, **kwargs):
         m.pop(field, None)
     for field in IGNORE_OPTIONS:
         m.get('options', {}).pop(field, None)
-    option_defaults = (DATADOG_DEFAULT_OPTIONS.items() +
-                       CONFIG['default_rule_options'].items())
+    option_defaults = (list(DATADOG_DEFAULT_OPTIONS.items()) +
+                       list(CONFIG['default_rule_options'].items()))
     for (field, value) in option_defaults:
         if m.get('options', {}).get(field) == value:
             del m['options'][field]
-    for (field, value) in CONFIG['default_rules'].items():
+    for (field, value) in list(CONFIG['default_rules'].items()):
         if m.get(field) == value:
             del m[field]
     # If options is {'thresholds': {'critical': x}}, then it is redundant.
     if not m.get('options'):
         m.pop('options', None)
-    elif m['options'].keys() == ['thresholds'] and m['options']['thresholds'].keys() == ['critical']:
+    elif list(m['options'].keys()) == ['thresholds'] and list(m['options']['thresholds'].keys()) == ['critical']:
         del m['options']
     m['name'] = m['name'].strip()
     original_team = original.get('team')
     team = original_team if original_team is not None else default_team
     severity = original.get('severity') or 'CRITICAL'
     if team:
-        if isinstance(team, basestring):
+        if isinstance(team, str):
```

Comment:
Sorry, I completely overlooked that. I pushed another commit in the meanwhile, which should fix that.